README.md

```diff
@@ -123,17 +117,17 @@ We use `quobyte` namespace for the examples. Create the namespace
   ```
 
 Quobyte requires a secret to authenticate volume create and delete requests. Create this secret with
- your Quobyte API login credentials. Kubernetes requires base64 encoding for secret data which can be obtained
- with the command `echo -n "value" | base64`. Please encode your user name and password in base64 and
- update [example/csi-secret.yaml](example/csi-secret.yaml)
+ your Quobyte API login credentials (Kubernetes requires base64 encoding for secret data which can be obtained
+ with the command `echo -n "value" | base64`). Please encode your user name, password (and access key
+ information) in base64 and update [example/csi-secret.yaml](example/csi-secret.yaml)
 
   ```bash
   kubectl create -f example/csi-secret.yaml
   ```
 
 Create a [storage class](example/StorageClass.yaml) with the `provisioner` set to `csi.quobyte.com` along with other configuration
  parameters. You could create multiple storage classes by varying `parameters` such as
-  `quobyteTenant`, `quobyteConfig` etc.
+  `quobyteTenant`, `quobyteConfig` etc. If you're using `Quobyte access keys`, make sure to configure the `user:`, `group:` and `accessMode:` for the volume as required.  
```

Comment:
Elaborate, not sure what that means.

Reply:
It is not required. Access keys will not change this behavior.